app/views/devise/registrations/edit.html.erb

```diff
@@ -8,7 +8,7 @@
     <%= f.email_field :email, autofocus: true %>
   </div>
 
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+  <% if devise_mapping.confirmable? && resource.can_send_reconfirmation? %>
```

Comment:
Same thing here - the `can_send_reconfirmation?` shouldn't replace `pending_reconfirmation?` on controllers and views.